pkg/cluster/install.go

```diff
@@ -226,6 +226,8 @@ func (m *manager) Update(ctx context.Context) error {
 		steps.Action(m.configureAPIServerCertificate),
 		steps.Action(m.configureIngressCertificate),
 		steps.Action(m.renewMDSDCertificate),
+		steps.Action(m.fixUserAdminKubeconfig),
```

Comment:
Nit: If you dig into this function, you can see that it strictly does a subset of what `generateKubeconfigs` does, so I don't think we need both here.